candidates/tests/auth.py

```diff
@@ -23,11 +23,14 @@ def setUpClass(cls):
                 ('ermintrude', 'user_who_can_rename', [TRUSTED_TO_RENAME_GROUP_NAME]),
                 ('frankie', 'user_who_can_record_results', [RESULT_RECORDERS_GROUP_NAME]),
         ):
-            u = User.objects.create_user(
-                username,
-                username + '@example.com',
-                'notagoodpassword',
-            )
+            try:
+                u = User.objects.get(username=username)
+            except User.DoesNotExist:
```

Comment:
It clearly can't do any harm, so best to keep it in! (I think I may have messed up my attempts to reproduce in a stupid way.)